puck.config.tsx

```diff
@@ -10,7 +10,7 @@ import { Video } from "components/Video";
 import { Root } from "./components/Root";
 import { Typography } from "./components/Typography";
 import { Divider } from "components/Divider";
-
+import { List } from "components/List";
```

Comment:
**praise:** We should have empty line after imports (but this handles Prettier, do you have it working?).

Reply:
It been working. I just didn't save the file before commiting.